ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -78,6 +80,10 @@ export default class StatusDropdown extends React.Component {
       filteredAlerts,
       frameworks,
     } = this.props;
+    const alertsWithVideos = new FilterAlertsWithVideos(
```

Comment:
We need to do the same thing in `copySummary()`. I think we should move this enrichment in `componentDidMount()` & enrich our alert summary there.

Reply:
This way we do the same thing I proposed: enrich the summaries when StatusDropdown is rendered which might overload the summary alerts page, not when `File bug` is button is pressed.